02-inspect-file.md

```diff
@@ -169,7 +169,7 @@ Instead of simply counting the files that match the search criteria, save them t
 This is done with the redirection operator, `>`.
 
 ~~~ {.bash}
-gunzip -c counts-raw.txt.gz | grep "Evolutionary Biology" | grep "Cell Biology" > evo-cell-bio.txt
+zcat counts-raw.txt.gz | cut -f11 | grep "Evolutionary Biology" | grep "Cell Biology" > evo-cell-bio.txt
```

Comment:
Omitting `cut` here was purposeful. We want to have the full data set so we can see which papers correspond to these terms, but if we use `cut` we lose all the other columns.

Thus the reason that `evo-cell-bio.txt` has 170 rows instead of only 153 is because some other columns must have contained the strings "Evolutionary Biology" or "Cell Biology". Likely either the column with the titles or the column with the subSubject tags.

This also shows the limitations of these Unix commands. We can get counts, but can't actually filter the data (we'd have to use something more sophisticated like `awk`). This leads into why R is necessary for further exploration.